backend/api.py

```diff
@@ -1,12 +1,14 @@
 from fastapi import FastAPI
 from fastapi.responses import JSONResponse
+from utils import Config
 from director import question
 from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
+config = Config()
 
 origins = [
-    "http://localhost:8650",
+    "http://localhost:" + config.port,
```

Comment:
Should the host also be part of the configuration. We're only running the UI locally for now but eventually the UI may  be hosted elsewhere (such as Azure).

So we'd have....
```
origins = [
    config.host + ":" + config.port,
]
```

Reply:
Alternatively, we stick to one `config.host` variable and that holds the entire URL including the port?

Reply:
I've updated the code to have a `config.frontend_url` instead and also made some of the names more consistent